tests/unit/dataactbroker/utils.py

```diff
@@ -0,0 +1,17 @@
+def add_models(db, models=None):
+    if models is None:
+        models = []
+
+    # Needs to happen in this order due to foreign key constraints between User and Submission
+    for model in models:
+        db.session.add(model)
+        db.session.commit()
```

Comment:
This feels a little kludgey; can we do something like
```python
submission.user = user
session.add(submission)
session.commit()
```
and expect the user to be added transitively?

If not, maybe we should rename this function to something like `add_models_in_order` to indicate how it's different.